lldb/test/API/functionalities/process_save_core_minidump/main.cpp

```diff
@@ -1,6 +1,7 @@
 #include <cassert>
 #include <iostream>
 #include <thread>
+thread_local size_t lf = 42;
```

Comment:
To be sure thread-locals work properly, I'd suggest also creating a shared library (which is dlopened at runtime) and creating a non-main thread, which will be used to access the value of the variable.